src/selectors/__tests__/skip-logic.test.js

```diff
@@ -2,6 +2,10 @@
 
 import { getNextIndex, isStageSkipped } from '../skip-logic';
 
+import { stages as getStages } from '../session';
+
+jest.mock('../session');
```

Comment:
The tests here make assertions about stages length (hardcoded). I think it's appropriate to mock the session selector here, but if we want integration tests between session & skip-logic, then I can revise.